TOPdeskPS/functions/Supporting Files/Get-TdPermissionGroup.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Get-TdPermissionGroup {
+    <#
+.SYNOPSIS
+    returns list of permission groups
+.DESCRIPTION
+    returns list of permission groups
+.EXAMPLE
+    PS C:\> Get-TdPermissionGroup
+    returns list of permission groups
+#>
+[CmdletBinding(HelpUri = 'https://andrewpla.github.io/TOPdeskPS/commands/Get-TdPermissionGroup')]
+
+    param
+    (
+
```

Comment:
add name parameter to filter on GroupName

Reply:
(y)